apps/i18n/music/en_us.json

```diff
@@ -1,4 +1,53 @@
 {
+  "blockly_blockNewTrackAtStart": "new track",
```

Comment:
Do we need to localize these non-`simple2` models?

Reply:
I went ahead and did this just to be safe, just in case we do end up pulling blocks from other modes into our current model